src/Shell/Task/QueueTask.php

```diff
@@ -30,20 +30,6 @@ abstract class QueueTask extends Shell implements QueueTaskInterface
      */
     public $QueuedTasks;
 
-    /**
-     * Timeout for run, after which the Task is reassigned to a new worker.
-     *
-     * @var int
-     */
-    public $timeout = 120;
-
-    /**
-     * Number of times a failed instance of this task should be restarted before giving up.
-     *
-     * @var int
-     */
-    public $retries = 1;
-
```

Comment:
Where do the defaults come from (now)?

Reply:
From Config.php defaultWorkerTimeout() and defaultWorkerRetries(). Value is found by:

1. check task for a value (if property exists, use that)
2. check config file (Queue.defaultWorkerTimeout and Queue.defaultWorkerRetries)
3. default to had coded value in Config.php (2 minutes, 4 retries)